packages/firestore/exp/src/api/database.ts

```diff
@@ -326,12 +266,12 @@ export function enableMultiTabIndexedDbPersistence(
  * but the client remains usable.
  */
 function setPersistenceProviders(
-  firestore: FirebaseFirestore,
+  firestore: FirestoreClient,
```

Comment:
This instance is no longer a Firestore or FirestoreCompat: rename to `client` or similar?

Reply:
Done